addons/ondevice-knobs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@storybook/addon-ondevice-knobs",
+  "version": "4.0.0-alpha.20",
+  "description": "Display storybook story knobs on your deviced.",
+  "keywords": [
+    "addon",
+    "knobs",
+    "ondevice",
+    "react-native",
+    "storybook"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storybooks/storybook.git"
+  },
+  "license": "MIT",
+  "main": "dist/index.js",
+  "jsnext:main": "src/index.js",
+  "scripts": {
+    "prepare": "node ../../scripts/prepare.js"
+  },
+  "dependencies": {
+    "@storybook/addons": "4.0.0-alpha.20",
+    "@storybook/addon-knobs": "4.0.0-alpha.20",
```

Comment:
didn't get where this in use

Reply:
A person who uses ondevice-knobs will be still importing addon-knobs inside their app. So if he doesn't install it directly we still want him to have it in his node_modules. @igor-dv.

Reply:
Yes, but users will need to install it explicitly. In that case, you probably want it to be a peer

Reply:
Alright, thanks :)